packages/rum/test/e2e/manual-timing/app.js

```diff
@@ -33,7 +33,11 @@ const apm = createApmBase({
   operationMode: 'manual'
 })
 
-const transaction = apm.startTransaction('transaction-name', 'transaction-type')
+const transaction = apm.startTransaction(
+  'transaction-name',
+  'transaction-type',
+  { managed: true }
```

Comment:
The test was to check for manual type right? can you explain why we need this here now?

Reply:
I think we forgot to update this test when we introduced the managed option, if you look at the rest of the code, we actually expect the that the xhrs to be captured by this transaction. So the test is for manual but managed transactions. we might need another test for manual but unmanaged transactions as well, it's out of the scope of this PR though.